src/main/java/net/sf/jabref/gui/GroupAddRemoveDialog.java

```diff
@@ -51,16 +57,16 @@ public GroupAddRemoveDialog(BasePanel panel, boolean add, boolean move) {
     @Override
     public void action() throws Throwable {
         GroupTreeNode groups = panel.getBibDatabaseContext().getMetaData().getGroups();
-        if (groups == null) {
+        if (Objects.isNull(groups)) {
```

Comment:
Objects.isNull exists more or less just to be used as a predicate (e.g. filter(Objects::isNull). The previous version with == null is more readable in general.